api/v1alpha1/mirrorpeer_types.go

```diff
@@ -20,26 +20,38 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// StorageClusterRef holds a reference to a StorageCluster
+type StorageClusterRef struct {
+	Name      string `json:"name"`
+	Namespace string `json:"namespace"`
+}
+
+// PeerRef holds a reference to a mirror peer
+type PeerRef struct {
+	// ClusterName is the name of ManagedCluster.
```

Comment:
PeerRef doesn't need any ID or name?

Reply:
Is there a need to uniquely identify each peer?